h2o-bindings/bin/gen_R.py

```diff
@@ -276,7 +277,7 @@ def gen_module(schema, algo, module):
         yield "  }"
         yield " "
         yield "  if (!missing(metalearner_params))"
-        yield "      parms$metalearner_params <- as.character(toJSON(metalearner_params, pretty = TRUE))"
+        yield "      parms$metalearner_params <- metalearner_params_json <- as.character(toJSON(metalearner_params, pretty = TRUE))"
```

Comment:
Why the double assignment?

Reply:
Because I need that original variable (it is later modified by other code) so I also copied the value into a separate variable.